### DISCUSSION TITLE
Wildcards in redirect URLs

### DISCUSSION THREAD

**Discussion by sdr0x07b6** [supabase/supabase#18128] [Questions] [answered]
Supabase > Authentication > URL Configuration > Redirect URLs
has this guide.

>URLs that auth providers are permitted to redirect to post authentication. Wildcards are allowed, for example, >https://*.domain.com

On my site, I would like to have the password set on this URL page in case of forgotten passwords.
http://localhost:3000/settings/personal/password

So even I open everything up like "http://localhost:3000/*", the confirmation email will not take you to /settings/personal/password This page. /login will open.
I add "http://localhost:3000/settings/personal/password" without the wildcard, the destination in the confirmation email will be the password page.

Are wildcards only valid for subdomains like in the guide message?
I can't confirm this as I don't currently have the environment to prepare a subdomain, but it seems that at least for the forgotten password configuration page, the wildcard can't open it up.

**Comment by GaryAustin1** [comment, marked as answer]
Try using ** instead of * as that is limited.
![image](https://github.com/supabase/supabase/assets/54564956/5d646a80-cbf0-4ca5-9412-14a620b932ef)

**Reply by GaryAustin1** [reply]
Also note the warning on using that in production...
https://supabase.com/docs/guides/auth/concepts/redirect-urls

**Reply by sdr0x07b6** [reply]
`**` worked as expected. thank you.
I didn't realize there was documentation that told me. This is a very useful page,
Authentication > URL Configuration > Redirect URLs
I think it would be even better for many users if there was a link in the guide message!
